src/MockClassifier.UnitTests/Services/Dmr/DmrServiceTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Microsoft.Extensions.Logging;
+using MockClassifier.Api.Services.Dmr;
+using MockClassifier.UnitTests.Extensions;
+using Moq;
+using RichardSzalay.MockHttp;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace MockClassifier.UnitTests.Services.Dmr
+{
+    public class DmrServiceTests
+    {
+        private static readonly DmrServiceSettings DefaultServiceConfig = new DmrServiceSettings
+        {
+            DmrApiUri = "https://dmr.fakeurl.com"
```

Comment:
I think since it is already in a `static readonly` field and there's only 1 place that it is being referenced it is effectively a const already. If that's okay with you, I'll keep it as is? 

Please let me know @martinkearn

Reply:
I'm seeing this URL repeated on lines 18 and 83. This is super-nit though, happy for you to leave it and resolve the thread.

Reply:
I totally missed that! Good spot. I've adjusted the code on Line 83 to use this so there's only one instance of the string. Thanks!